src/dash-adapter.js

```diff
@@ -221,6 +221,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
       }
       if (abr.restrictions) {
         if (abr.restrictions.minBitrate > 0) {
+          adapterConfig.capLevelToPlayerSize = false;
```

Comment:
Same here, this is by default false.
If an app set this and wants to apply other means of capping it needs to set it to false by itself.

Reply:
I did the same as we did for maxBitrate a few lines after I just wanted to keep the logic, once the restrictions set it'll change it to false.